finish/src/main/java/io/openliberty/guides/bulkhead/global/eBank/microservices/Utils.java

```diff
@@ -11,9 +11,10 @@
 package io.openliberty.guides.bulkhead.global.eBank.microservices;
 
 
-
 public class Utils {
 
+    // default timeout in the microservice
+    public static final int TIMEOUT = 20000;
```

Comment:
wondering why the TIMEOUT is put in the Utils file instead of in the ChatSession.java.

The comment should be 
// The length of time (in milliseconds) to pause the currently executing thread so as to simulate concurrent
// VFA chat requests